libs/slides/src/lib/deck/deck.component.ts

```diff
@@ -51,18 +54,40 @@ export class SlidesDeckComponent {
   }
 
   nextSlide() {
-    this.goToSlide(this.activeSlideIndex + 1);
+    if (this.activeSlideIndex + 1 < this.slides.length) {
+      this.goToSlide(this.activeSlideIndex + 1);
+    } else if (this.nextLink) {
+      this.router.navigateByUrl(this.nextLink);
+    }
   }
 
   previousSlide() {
-    this.goToSlide(this.activeSlideIndex - 1);
+    if (this.activeSlideIndex > 0) {
+      this.goToSlide(this.activeSlideIndex - 1);
+    } else if (this.previousLink) {
+      this.router.navigateByUrl(this.previousLink);
+    }
   }
 
   canGoNext(): boolean {
-    return this.activeSlideIndex + 1 < this.slides.length;
+    return (
+      this.activeSlideIndex + 1 < this.slides.length ||
+      (this.nextLink != null && this.nextLink !== '')
+    );
   }
 
   canGoPrevious(): boolean {
-    return this.activeSlideIndex > 0;
+    return (
+      this.activeSlideIndex > 0 ||
+      (this.previousLink != null && this.previousLink !== '')
```

Comment:
just this.previousLink?
also the same above

Reply:
tried using this.previousLink and (this.previousLink) but it was giving an error. i think it is because it is from a return statement... i would prefer to avoid the 2 checks...